source/bdDetect.py

```diff
@@ -195,8 +252,16 @@ def _getStandardHidDriverName() -> str:
 	return brailleDisplayDrivers.hidBrailleStandard.HidBrailleDriver.name
 
 
+def _isHIDUsagePageMatch(match: DeviceMatch, usagePage: int) -> bool:
+	return match.type == ProtocolType.HID and match.deviceInfo.get("HIDUsagePage") == usagePage
+
+
 def _isHIDBrailleMatch(match: DeviceMatch) -> bool:
-	return match.type == DeviceType.HID and match.deviceInfo.get("HIDUsagePage") == HID_USAGE_PAGE_BRAILLE
+	return _isHIDUsagePageMatch(match, HID_USAGE_PAGE_BRAILLE)
+
+
+def HIDUsagePageMatchFuncFactory(usagePage: int) -> MatchFuncT:
```

Comment:
should this be private? if not, please add a docstring

Reply:
Doc string added.